system/factory_manager.py

```diff
@@ -1,6 +1,7 @@
 # coding=utf-8
 import signal
 
+from django.utils import importlib
```

Comment:
Lol, whoops, will get it soon
On 13 Dec 2015 18:43, "Sean Gordon" <notifications@github.com> wrote:

> In system/factory_manager.py
> <https://github.com/UltrosBot/Ultros/pull/91#discussion_r47448227>:
>
> > @@ -1,6 +1,7 @@
> >  # coding=utf-8
> >  import signal
> >
> > +from django.utils import importlib
>
> Using PyCharm's "Add import" tool without actually looking at the options?
> :P
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UltrosBot/Ultros/pull/91/files#r47448227>.
>